### DISCUSSION TITLE
v0.16.0

### DISCUSSION THREAD

**Discussion by StevenACoffman** [99designs/gqlgen#1843] [Announcements]
## What's Changed
* Bump gqlgen.com version list by @ipfans in https://github.com/99designs/gqlgen/pull/1778
* Imporve gqlgen test cases by @ipfans in https://github.com/99designs/gqlgen/pull/1773
* Fix #1834: Implement federation correctly by @newerbie in https://github.com/99designs/gqlgen/pull/1835
* Update README.md by @patriotphoenix in https://github.com/99designs/gqlgen/pull/1836
* Fix #1832 @requires directive when @entityResolver is used by @chedom in https://github.com/99designs/gqlgen/pull/1833
* Breaking: Revert "Breaking: Fix plugin addition" by @Code-Hex in https://github.com/99designs/gqlgen/pull/1838
* add PrependPlugin by @Code-Hex in https://github.com/99designs/gqlgen/pull/1839
* Prepare for v0.16.0 release by @StevenACoffman in https://github.com/99designs/gqlgen/pull/1842

## New Contributors
* @newerbie made their first contribution in https://github.com/99designs/gqlgen/pull/1835
* @patriotphoenix made their first contribution in https://github.com/99designs/gqlgen/pull/1836
* @chedom made their first contribution in https://github.com/99designs/gqlgen/pull/1833

**Full Changelog**: https://github.com/99designs/gqlgen/compare/v0.15.1...v0.16.0

<hr /><em>This discussion was created from the release <a href='https://github.com/99designs/gqlgen/releases/tag/v0.16.0'>v0.16.0</a>.</em>

**Comment by StevenACoffman** [comment]
Hey folks! Please note that if you relied on the plugin order behavior in 0.14.0 in `AddPlugin`, this release restores that. If, instead, you want the order to go the other way, use `PrependPlugin` instead. This is potentially breaking so I wanted to highlight it.

**Comment by hansgylling** [comment]
Thanks for doing releases and getting this project going again after the long break. It's appreciated.

**Comment by softshipper** [comment]
Thanks a lot guys..
